rca/enquire_to_study/templates/enquire_to_study/index.html

```diff
@@ -1,4 +1,4 @@
-{% extends "wagtailadmin/base.html" %}
+{% extends "modeladmin/index.html" %}
```

Comment:
This seems to be the correct template to extend: https://github.com/wagtail/wagtail/blob/main/wagtail/contrib/modeladmin/templates/modeladmin/index.html

Reply:
Great work @katdom13